src/ui-actions/new-folder-conversation.tsx

```diff
@@ -0,0 +1,373 @@
+/*
```

Comment:
Why is this file commented?

Reply:
@nubsthead I don't remember correctly but I think I was implementing move to new Folder action inside the move modal and maybe I forgot to do it :XD